File: src/main/java/tech/zolhungaj/amqapi/servercommands/social/FriendRequestReceived.java
package tech.zolhungaj.amqapi.servercommands.social;

import com.squareup.moshi.Json;
import tech.zolhungaj.amqapi.servercommands.CommandType;

@CommandType("new friend request recived")//sic
public record FriendRequestReceived(
        @Json(name = "name") String playerName
){}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/social/FriendRequestReceived.kt
package tech.zolhungaj.amqapi.servercommands.social

import com.squareup.moshi.Json
import tech.zolhungaj.amqapi.servercommands.CommandType

@JvmRecord
@CommandType("new friend request recived") //sic
data class FriendRequestReceived(
    @Json(name = "name")
    val playerName: String
)
